plugins/webhooks/index.js

```diff
@@ -1,6 +1,6 @@
 'use strict';
 
-const githubRoute = require('./github');
+const webhooksRoute = require('./webhooks');
```

Comment:
Why name this webhooks?  Why not just merge the code in `github.js` into here?  Technically this is actually an ScmHook